javalanglib/src/main/scala/java/lang/Math.scala

```diff
@@ -102,11 +102,11 @@ object Math {
     if (a != a || a == PositiveInfinity)
       a
     else if (a == NegativeInfinity)
-      -MaxValue
+      MinValue
```

Comment:
Why this change? The documentation says that it should be `-MaxValue`. Have tested it on the JVM?